openedx_webhooks/github_views.py

```diff
@@ -46,29 +49,34 @@ def pull_request():
         return "PONG"
 
     pr = event["pull_request"]
+    PR_NUMBER = pr['number']
```

Comment:
It's a dynamically valued constant within the scope of the function, since it's *not* to be changed for the duration of the function execution. Again, if you feel strongly about it, I can change it.